build.gradle.kts

```diff
@@ -23,9 +23,13 @@ repositories {
 dependencies {
     implementation(group = "org.junit.jupiter", name = "junit-jupiter-api", version = "5.4.2")
     implementation(group = "org.junit.jupiter", name = "junit-jupiter-params", version = "5.4.2")
+    implementation(group = "org.junit.platform", name = "junit-platform-launcher", version = "1.4.2")
+    // For Java 8 provided would be enough as its part of the JRE,
+    // but Java 11 and 14 need this dependency cause of Jigsaw
+    implementation(group = "javax.xml.bind", name ="jaxb-api", version = "2.3.1")
+    implementation(group = "com.sun.xml.bind", name ="jaxb-impl", version = "2.3.3")
 
     testImplementation(group = "org.junit.jupiter", name = "junit-jupiter-engine", version = "5.4.2")
-    testImplementation(group = "org.junit.platform", name = "junit-platform-launcher", version = "1.4.2")
     testImplementation(group = "org.junit.platform", name = "junit-platform-testkit", version = "1.4.2")
```

Comment:
i would not split the dependencies on junit platfrom. i think , that same packages should stick together. never the less, if they are used for test or implementation. It makes upgrading easier, because the version which needs to be changed are closer to each other

Reply:
I moved the dependency, because it has another scope.